pkg/services/auth/jwt/plugin_auth.go

```diff
@@ -91,11 +93,16 @@ func (s *pluginAuthService) Verify(ctx context.Context, token string) (models.JW
 	return Verify(ctx, s.keySet, expectedClaims, additionalClaims, token)
 }
 
-func (s *pluginAuthService) Generate(usr *user.SignedInUser, tenantID int64, audience string) (string, error) {
+func (s *pluginAuthService) Generate(usr *user.SignedInUser, audience string) (string, error) {
 	if !s.IsEnabled() {
 		return "", errors.New("JWT token generation is disabled")
 	}
 
+	tenantID, err := strconv.ParseInt(os.Getenv("stackID"), 10, 64)
```

Comment:
not a big deal, but i was thinking this should be passed into the function to make testing easier